common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -0,0 +1,66 @@
+package views.support.cleaner
+
+import com.gu.contentapi.client.model.v1.{Content => ApiContent}
+import model.{Article, Content}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.scalatest.{FlatSpec, Matchers}
+
+class AmpEmbedCleanerTest extends FlatSpec with Matchers {
+
+  private def clean(document: Document): Document = {
+    val cleaner = AmpEmbedCleaner(article())
```

Comment:
Nit: you can pass a html string as argument and do the `Jsoup.parse()` in this method rather than repeating it in every single tests :)